test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -123,10 +128,31 @@ func SaturateZoneUntilLeft(zone nrtv1alpha1.Zone, requiredRes corev1.ResourceLis
 	return paddingRes, nil
 }
 
+func SaturateNodeUntilLeft(node corev1.Node, requiredRes corev1.ResourceList) (corev1.ResourceList, error) {
```

Comment:
It seems we will need to rename functions here in a future cleanup PR, because the `Saturate*` function family is not saturating a node, but rather computing the resources needed to saturate a node.

Reply:
makes sense.